### PR TITLE
chore: add wagmi sdk

### PR DESCRIPTION
**Pull request by guanbinrui**
## Description

Closes MF-0000

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/injected-script/main/WagmiBridge/config.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import type { Chain, FallbackTransport, PublicClient } from 'viem'
+import { arbitrum, aurora, avalanche, bsc, fantom, gnosis, mainnet, optimism, polygon } from 'viem/chains'
+import { configureChains, createConfig, type Config, type WebSocketPublicClient } from '@wagmi/core'
+import { publicProvider } from '@wagmi/core/providers/public'
```
Comment (Jack-Works) [inline packages/injected-script/main/WagmiBridge/config.ts]: don't import them at the top level. import them with dynamic import.

File: packages/injected-script/main/WagmiBridge/config.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+)
+
+export const config: Config<
+    PublicClient<FallbackTransport, Chain>,
+    WebSocketPublicClient<FallbackTransport>
+> = createConfig({
+    autoConnect: true,
+    publicClient,
+    webSocketPublicClient,
+})
```
Comment (Jack-Works) [inline packages/injected-script/main/WagmiBridge/config.ts]: don't create them until needed.

File: packages/injected-script/main/WagmiBridge/index.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import { connect, disconnect, getNetwork } from '@wagmi/core'
+import { MetaMaskConnector } from '@wagmi/core/connectors/metaMask'
+import { WalletConnectConnector } from '@wagmi/core/connectors/walletConnect'
+import { config } from './config.js'
+import { handlePromise } from '../utils.js'
+
+config.subscribe((state) => {
+    console.log('DEBUG: state')
+    console.log(state)
+})
```
Comment (Jack-Works) [inline packages/injected-script/main/WagmiBridge/index.ts]: don't subscribe until needed

File: packages/injected-script/main/intrinsic_brand.ts [hunk 4]
```diff
@@ -1,4 +1,5 @@
-import { $, $safe } from './intrinsic.js'
+import * as $ from './intrinsic_content.js'
+import * as $safe from './intrinsic_blessed.js'
```
Comment (Jack-Works) [inline packages/injected-script/main/intrinsic_brand.ts:2]: why change this?

File: packages/injected-script/main/intrinsic_unsafe.ts [hunk 5]
```diff
@@ -1,5 +1,6 @@
 import { takeThisF } from './intrinsic_content.js'
-import { noop } from './utils.js'
+
+function noop() {}
```
Comment (Jack-Works) [inline packages/injected-script/main/intrinsic_unsafe.ts:3]: why change this?

File: packages/injected-script/main/utils.ts [hunk 6]
```diff
@@ -1,5 +1,7 @@
 import { CustomEventId, encodeEvent, type InternalEvents } from '../shared/index.js'
-import { $unsafe, $, $safe } from './intrinsic.js'
+import * as $ from './intrinsic_content.js'
+import * as $unsafe from './intrinsic_unsafe.js'
+import * as $safe from './intrinsic_blessed.js'
```
Comment (Jack-Works) [inline packages/injected-script/main/utils.ts:4]: why change this?

**Review by Jack-Works — CHANGES_REQUESTED** [review]

File: packages/injected-script/main/intrinsic_brand.ts (hunk 4 above)
Comment (guanbinrui) [reply packages/injected-script/main/intrinsic_brand.ts:2]: Circular dep.

File: packages/injected-script/main/utils.ts (hunk 6 above)
Comment (guanbinrui) [reply packages/injected-script/main/utils.ts:4]: Circular dep.

File: packages/injected-script/main/intrinsic_unsafe.ts (hunk 5 above)
Comment (guanbinrui) [reply packages/injected-script/main/intrinsic_unsafe.ts:3]: To ease circular dep.

File: packages/injected-script/main/WagmiBridge/config.ts (hunk 1 above)
Comment (guanbinrui) [reply packages/injected-script/main/WagmiBridge/config.ts]: I will do it after everything works fine.

**Pull request merged**